raw_radar_reader/README.md

```diff
@@ -0,0 +1,72 @@
+# Raw Radar Interface
+
+This directory contains tools for interacting with raw radar data. Zendar
+makes available raw ADC samples from our FMCW radars. The example tools
+in this directory allow researchers to easily use this data format.
+
+o## Dependencies
+```
+libprotobuf
+protobuf-compiler
+```
+
+Python packages:
+```
+numpy
+matplotlib
+gzip
+os
+struct
+protobuf
+argparse
+```
```

Comment:
Shouldn't we provide installation steps or a ```requirements.txt``` file to help users install these packages? also, are there certain version numbers that users should install?

On my laptop machine (which is not setup for building ```ZenCode```, I get the following errors when trying to install gzip, os, and struct:

```
/home/osama/Code/ZendarSDK/raw_radar_reader/protocol>pip3 install gzip 
Collecting gzip
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 353, in run
    wb.build(autobuilding=True)
  File "/usr/lib/python3/dist-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 554, in _prepare_file
    require_hashes
  File "/usr/lib/python3/dist-packages/pip/req/req_install.py", line 278, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 465, in find_requirement
    all_candidates = self.find_all_candidates(req.name)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 423, in find_all_candidates
    for page in self._get_pages(url_locations, project_name):
  File "/usr/lib/python3/dist-packages/pip/index.py", line 568, in _get_pages
    page = self._get_page(location)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 683, in _get_page
    return HTMLPage.get_page(link, session=self.session)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 795, in get_page
    resp.raise_for_status()
  File "/usr/share/python-wheels/requests-2.18.4-py2.py3-none-any.whl/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://pypi.org/simple/gzip/
/home/osama/Code/ZendarSDK/raw_radar_reader/protocol>pip3 install os
Collecting os
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 353, in run
    wb.build(autobuilding=True)
  File "/usr/lib/python3/dist-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 554, in _prepare_file
    require_hashes
  File "/usr/lib/python3/dist-packages/pip/req/req_install.py", line 278, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 465, in find_requirement
    all_candidates = self.find_all_candidates(req.name)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 423, in find_all_candidates
    for page in self._get_pages(url_locations, project_name):
  File "/usr/lib/python3/dist-packages/pip/index.py", line 568, in _get_pages
    page = self._get_page(location)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 683, in _get_page
    return HTMLPage.get_page(link, session=self.session)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 795, in get_page
    resp.raise_for_status()
  File "/usr/share/python-wheels/requests-2.18.4-py2.py3-none-any.whl/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://pypi.org/simple/os/
/home/osama/Code/ZendarSDK/raw_radar_reader/protocol>pip3 install struct
Collecting struct
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 353, in run
    wb.build(autobuilding=True)
  File "/usr/lib/python3/dist-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 554, in _prepare_file
    require_hashes
  File "/usr/lib/python3/dist-packages/pip/req/req_install.py", line 278, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 465, in find_requirement
    all_candidates = self.find_all_candidates(req.name)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 423, in find_all_candidates
    for page in self._get_pages(url_locations, project_name):
  File "/usr/lib/python3/dist-packages/pip/index.py", line 568, in _get_pages
    page = self._get_page(location)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 683, in _get_page
    return HTMLPage.get_page(link, session=self.session)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 795, in get_page
    resp.raise_for_status()
  File "/usr/share/python-wheels/requests-2.18.4-py2.py3-none-any.whl/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://pypi.org/simple/struct/
```

Reply:
Thanks for catching that, I added a requirements.txt. 

The packages you had trouble with are part of standard python, I neglected to prune them from my dependency list.

Reply:
Can you try again?

Reply:
OK, here's what I'm getting right now after ignoring the packages I mentioned above:

```
/home/osama/Code/ZendarSDK/raw_radar_reader>python3 ./python/example.py --root ~/zen-ffa/nyquist/2020-12-04_0048/000 --calibration ~/zen-ffa/nyquist/2020-12-04_0048/000
Traceback (most recent call last):
  File "./python/example.py", line 5, in <module>
    from acquisition import AcquisitionStreamModel
  File "/home/osama/Code/ZendarSDK/raw_radar_reader/python/acquisition.py", line 5, in <module>
    from acquisition_pb2 import (
ModuleNotFoundError: No module named 'acquisition_pb2'
```
Looks like I have built this file under the protocol directory:
```
/home/osama/Code/ZendarSDK/raw_radar_reader>find . | grep -i acquisition_pb2
./protocol/acquisition_pb2.py
```

and my PYTHONPATH looks OK too:
```
/home/osama/Code/ZendarSDK/raw_radar_reader>env | grep protocol 
PYTHONPATH=:/home/osama/code/ZendarSDK/raw_radar_reader/protocol:/home/osama/code/ZendarSDK/raw_radar_reader/python
```

Did I miss something?

Reply:
I cannot reproduce your issue. I started from a new terminal, zeroed out my PYTHONPATH and copy-pasted the instructions in the README and the command runs through just fine. 

Are you sure that `/home/osama/code/ZendarSDK/raw_radar_reader/protocol` and `/home/osama/code/ZendarSDK/raw_radar_reader/python` exist?

Reply:
I just checked now, and these two directories exist on my machine. I also just retried your new ```requirements.txt``` file and the installation when smoothly this time. This seems to be resolved now.